complete/src/main/java/com/example/restservice/service/QueueService.java

```diff
@@ -78,9 +79,18 @@ public QueueStatusResponse getQueueStatus(String queueId) {
 
   @Transactional
   public MyQueuesResponse getMyQueues() {
-    return new MyQueuesResponse(
+    Stream<Queue> myQueueStream =
```

Comment:
can you do this without using this extra varaible?

Reply:
removed extra variable